packages/core/upload/server/controllers/content-api.js

```diff
@@ -1,19 +1,25 @@
 'use strict';
 
 const _ = require('lodash');
+const { omit, isArray, isPlainObject } = require('lodash/fp');
 const utils = require('@strapi/utils');
 const { getService } = require('../utils');
-const validateSettings = require('./validation/settings');
-const validateUploadBody = require('./validation/upload');
+const validateUploadBody = require('./validation/content-api/upload');
 
 const { sanitize } = utils;
 const { ValidationError } = utils.errors;
 
+const removeLocation = data => {
+  if (isArray(data)) return data.map(omit('location'));
+  if (isPlainObject(data)) return omit('location', data);
+  return data;
```

Comment:
What is the 3rd case that can happen here? `String`?

Reply:
It's default case and it handles when it's `null`